src/sass/_patterns-global-nav.scss

```diff
@@ -1,171 +1,307 @@
+$global-asset-path: 'https://assets.ubuntu.com/v1/';
+$navigaiton-height: 1.875rem;
+@media only screen and (max-width: $global-breakpoint-medium) {
+  $navigaiton-height: 2.5rem;
+}
+
 @mixin global-p-nav {
   body {
-    padding-top: 30px;
+    padding-top: $navigaiton-height;
+    position: relative;
+
+    @media only screen and (max-width: $global-breakpoint-medium) {
+      padding-bottom: $navigaiton-height;
+      padding-top: 0;
+    }
   }
 
   .global-nav {
     background: $global-background;
-    width: 100%;
-    height: 30px;
+    box-shadow: 0 2px 2px transparentize($global-black, .8);
     display: block;
-    z-index: 101;
-    position: absolute;
-    top: 0;
+    height: $navigaiton-height;
     left: 0;
-    box-shadow: 0 2px 2px rgba(0, 0, 0, .2);
     margin: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: 101;
+
+    @media only screen and (max-width: $global-breakpoint-medium) {
+      background: $global-transparent;
+      bottom: 0;
+      box-shadow: none;
+      height: $navigaiton-height;
+      top: auto;
+    }
 
     &__wrapper {
-      box-sizing: border-box;
       background: none repeat scroll 0 0 $global-white;
+      box-sizing: border-box;
       margin: 0 auto;
       position: relative;
       text-align: left;
-      width: 984px;
+      width: 100%;
 
-      @media only screen and (min-width: 768px) {
-        width: auto !important;
+      @media only screen and (min-width: $global-breakpoint-medium) {
+        width: auto;
       }
 
-      @media only screen and (max-width: 768px) {
-        width: 100%;
+      @media only screen and (min-width: $global-breakpoint-large) {
+        width: $global-breakpoint-large;
       }
+    }
+
+    &__title {
+      display: none;
+
+      @media only screen and (max-width: $global-breakpoint-medium) {
+        background-color: transparent;
+        background-image: url('#{$global-asset-path}7bd1bd7b-arrow_down_9fa097.svg');
+        background-position: calc(100% - .625rem) 50%;
+        background-repeat: no-repeat;
+        background-size: .8125rem;
+        border-bottom: 1px solid $global-mid-grey;
+        border-right: 0;
```

Comment:
Need to add ```border-left: 0``` as I can see it on small screen 
<img width="429" alt="screen shot 2017-05-03 at 10 28 06" src="https://cloud.githubusercontent.com/assets/1201618/25655949/ceba6654-2fee-11e7-8688-b9ca27d09e57.png">